### PR TITLE
Add codeowners file

### LINKED ISSUE #80: Create a CODEOWNERS file
We should create a CODEOWNERS file for the individual directories of the different languages. That way, the various teams will automatically be requested a review for when [dependabot](https://github.com/libp2p/universal-connectivity/issues/79) opens a new PR.

### PR DESCRIPTION
**Pull request by thomaseizinger**
Resolves #80.

### CODE REVIEW

**Comment by thomaseizinger** [comment]
Can somebody with admin rights deal with this?

image

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+rust-peer/*             @libp2p/rust-libp2p-maintainers
+go-peer/*               @libp2p/go-libp2p-maintainers
+packages/frontend/*     @libp2p/js-libp2p-chainsafe-maintainers
```
Comment (maschad) [inline .github/CODEOWNERS:3]: I think this  needs to be changed to `js-peer` and also the codeowner is `@libp2p/js-libp2p-dev` see the [js-libp2p CODEOWNERS file](https://github.com/libp2p/js-libp2p/blob/master/CODEOWNERS) for reference.

**Review by maschad — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (thomaseizinger) [reply .github/CODEOWNERS:3]:
```suggestion
js-peer/*     @libp2p/js-libp2p-dev 
```

**Comment by thomaseizinger** [comment]
An admin still needs to add the teams as maintainers to the repo.

**Comment by maschad** [comment]
Yes I've tagged @p-shahi as a reviewer, he should be able resolve that.

**Comment by thomaseizinger** [comment]
Friendly ping @p-shahi ! Can you look at this?

**Comment by p-shahi** [comment]
Hi @thomaseizinger I believe this should be done via github-mgmt, created a PR there: https://github.com/libp2p/github-mgmt/pull/185

**Pull request closed without merging**